modules/ossm-config-otel.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/traces/ossm-distr-tracing-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-otel_{context}"]
+= Configuring {OTELName} with Service Mesh
+
+You can integrate {SMProductName} with {OTELName} to instrument, generate, collect, and export telemetry traces, metrics, and logs to analyze and understand your software's performance and behavior.
+
+.Prerequisites
+
+* A TempoStack is installed and configured as described here: link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/distributed_tracing/distributed-tracing-platform-tempo#installing-a-tempostack-instance[Installing a TempoStack instance].
+* An instance of an `OpenTelemetry` collector is already configured in the `istio-system` namespace.
+* An Istio instance is created in the `Istio` resource (`Istio.spec.profile`).
+* An Istio CNI instance is created in the `Istio` resource (`Istio.spec.profile`).
```

Comment:
Same as above you don't need to mention `Istio.spec.profile` and `IstioCNI` is the resource.